### DISCUSSION TITLE
E.Limits

### DISCUSSION THREAD

**Discussion by RaB54** [ExpressLRS/ExpressLRS#3004] [Q&A]
## Summary

I want to activate E.Limits on my Radiomaster Boxer ELRS (EdgeTX 2.10.1; Packet Rate 250Hz).

When I tick the E.Limits box I can run the end points out to 150 but the servo does not run beyond 100. If I have 120 set in OUTPUTS - Ch1 but the servo only wnds to the position that equates to 100. It seems the system is recognisinb the limit of 120 but the command is not being sent to the Receiver.

I have discovered if I use a Packet Rate for 100Hz it seems to work.

Why is this active only on some Packet Rates?

**Comment by pkendall64** [comment]
It is active for full-res packet modes only.

**Comment by mha1** [comment]
> Why is this active only on some Packet Rates?

Because that's the way it was designed. Check out [ExpressLRS Doc](https://www.expresslrs.org/software/switch-config/?h=summary+switch+configuration+modes#summary-of-switch-configuration-modes) and take note of the (not so) subtle difference between "Normal Range" and "CRSF ext. Limits". As you can see extended limits are only supported in the full-res modes, i.e. 100Hz and 333Hz.
